Editor/GitLogWindow.cs

```diff
@@ -262,28 +331,35 @@ private void DrawFilesPanel(Module module, IEnumerable<string> selectedCommitHas
 
             using (new EditorGUILayout.HorizontalScope())
             {
-                if (selectedFiles != null)
-                {
-                    var panelSize = new Vector2(position.width - InfoPanelWidth, FilesPanelHeight);
-                    treeViewFiles.Draw(panelSize, new[] { diffFiles }, (_) => ShowFileContextMenu(module, selectedFiles, selectedCommitHashes.First()), (id) => SelectAsset(id, diffFiles));
-                }
-                else
-                {
-                    GUILayout.Space(position.width - InfoPanelWidth);
-                }
                 using (var scroll = new EditorGUILayout.ScrollViewScope(infoPanelScrollPosition))
                 using (new EditorGUILayout.VerticalScope(GUILayout.Height(FilesPanelHeight)))
                 {
                     string lastHash = selectedCommitHashes.Count() > 1 ? selectedCommitHashes.Last() : null;
                     EditorGUILayout.SelectableLabel($"{selectedCommitHashes.First()} {lastHash?.WrapUp("- ", "")}");
                     foreach (var selectedCommitHash in selectedCommitHashes)
                     {
-                        LogLine commitLine = lines?.FirstOrDefault(x => x.Hash == selectedCommitHash);
+                        GUILayout.BeginHorizontal();
+                        var commitLine = lines?.FirstOrDefault(x => x.Hash == selectedCommitHash);
+                        var avatar = GitUserBase.GetUserData(commitLine.Email, commitLine.Author).avatar;
+                        if (avatar)
+                            GUILayout.Box(avatar);
                         if (commitLine != null)
-                            EditorGUILayout.TextField(commitLine.Raw.AfterFirst('-'), CommitInfoStyle.Value, GUILayout.Height(60));
+                        {
+                            EditorGUILayout.TextField(commitLine.Raw.AfterFirst('-'), CommitInfoStyle.Value, GUILayout.Height(GitUserBase.AvatarSize));
+                        }
+                        GUILayout.EndHorizontal();
                     }
                     infoPanelScrollPosition = scroll.scrollPosition;
                 }
+                if (selectedFiles != null)
```

Comment:
Maybe return list of files to the right part? Because list of files is most important and I would prefer seeing it first.